release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -257,6 +257,14 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="egress-ip-etp-local-support"]
+=== Support for EgressIP with ExternalTrafficPolicy=Local and direct ExternalToPod routing
+
+Previously, the use of `ExternalTrafficPolicy=Local` was unsupported with EgressIP configurations when using MetalLB or an F5 load balancer. When attempting this configuration, connections reaching pods via the OVN-Kubernetes routing gateway were returned through EgressIP if an EgressIP was configured for a namespace, which affects how responses to incoming connections are handled. This led  to service disruption when `externalTrafficPolicy` was set to `local`. Consequently, this cause multiple issues for customers. For MetalLB, this resulted in inaccessible load balancer IPs and broke the expected behavior of the service. For F5 LBs, it disrupted traffic routing and led to dropped connections and service unavailability.
```

Comment:
```suggestion
Previously, the scenario when EgressIP selected pods were also backends to Services of type `ExternalTrafficPolicy=Local` was unsupported. When attempting this configuration, Service ingress traffic reaching pods were rerouted incorrectly to egress node hosting the EgressIP. This effected how responses to incoming service traffic connections are handled. This led  to non-functional service when `externalTrafficPolicy` was set to `local`since connections were dropped and service became unavailable.
```
something like this .. (FYI: I am not a native english speaker)